### ISSUE TITLE
purpose of project.json?

### ISSUE THREAD

**Issue by nhooyr** [buildkite/golang-example#5]
https://github.com/buildkite/golang-example/blob/b33574c9c44322ca3ce74622e804460c68b754a5/.buildkite/project.json

**Comment by toolmantim** [comment]
This helps power the "Add to Buildkite" button in the readme. Not something you'd typically need for your own projects, but if they're open source, and you want people to easily set up the repo in their own Buildkite account, then you can add one to your own repos too.

**Comment by nhooyr** [comment]
This behaviour could use better documentation.

**Comment by toolmantim** [comment]
You're absolutely right. Sorry that it was confusing. 

I just updated all the example repos to use a `template.yml` file instead, which allows for a comment to explain what it is, and I also added a note to the examples-pipelines index (https://github.com/buildkite/example-pipelines/commit/0d1d226fab50ece6963f5e02eeec7a2979656bc9) saying you don't need this file in your own projects.

Hope that helps, and sorry again for the confusion!
